app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HelloReactOnRails</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= javascript_include_tag "/assets/application.js", "data-turbo-track": "reload", defer: true %>
```

Comment:
- [ ] It would be better if we fix the routing error I get when I tried to run the app. Here is what i did;
- after cloning; I did `git checkout dev` to make sure I'm in the dev branch. Then I proceeded to check my database config file. Next I run `rails db:create db:migrate db:seed` although you have not seeded any data. It was successful. Next I had to run the server `rails s` and here is the result as shown in the screenshot
![Screenshot from 2022-11-22 13-21-15](https://user-images.githubusercontent.com/46388113/203326938-7f15f280-de02-47fd-898a-ff5140a317ad.png)

I'm unable to view the app

Reply:
Didn't you run npm install? Bro, you need to also run npm run build. What the fck guys? You guys wasted my 2 reviews.

Reply:
> Didn't you run npm install? Bro, you need to also run npm run build. What the fck guys? You guys wasted my 2 reviews.

I did, although you did not include it in your README file :slightly_smiling_face:I found the need to run the npm install command.  What I mean is that I did run the npm install and still got the same error. As code reviewers our job is to test the app to the best of our knowledge. Sorry about wasting your 2 reviews but I couldn't overlook it